docs/json-api.md

```diff
@@ -2129,6 +2129,7 @@ GET /api/search
 | artists         | object   | [`paging`](#paging-object) object containing [`artist`](#artist-object) objects that matches the `query` |
 | albums          | object   | [`paging`](#paging-object) object containing [`album`](#album-object) objects that matches the `query` |
 | playlists       | object   | [`paging`](#paging-object) object containing [`playlist`](#playlist-object) objects that matches the `query` |
+| composers       | object   | [`paging`](#paging-object) object containing `composers` objects that matches the `query` |
```

Comment:
There is no such thing as a "composers" object. I have changed to browse-info object with commit 3e7e03b4c18b091b01b66e62467067e7cbf50da4. I have also added support for genres, so that has also been re-added.